packages/next/build/output/log.ts

```diff
@@ -1,12 +1,12 @@
-import chalk from 'chalk'
+import * as chalk from 'chalk'
 
 const prefixes = {
-  wait: chalk`[ {cyan wait} ] `,
-  error: chalk`[ {red error} ]`,
-  warn: chalk`[ {yellow warn} ] `,
-  ready: chalk`[ {green ready} ]`,
-  info: chalk`[ {cyan {dim info}} ] `,
-  event: chalk`[ {magenta event} ]`,
+  wait: chalk.cyan('[ wait ] '),
```

Comment:
This changes the coloring of the output.

It should be `[ ${chalk.cyan('wait')} ]`

Same for the other messages